src/main/java/org/broadinstitute/consent/http/models/dataset_registration_v1/DatasetRegistrationSchemaV1UpdateValidator.java

```diff
@@ -20,6 +27,89 @@ public DatasetRegistrationSchemaV1UpdateValidator(DatasetService datasetService)
     this.datasetService = datasetService;
   }
 
+  private final ExclusionStrategy studyExclusionStrategy = new ExclusionStrategy() {
+    @Override
+    public boolean shouldSkipField(FieldAttributes fieldAttributes) {
+      return fieldAttributes.getName().equalsIgnoreCase("dataSubmitterUserId");
+    }
+
+    @Override
+    public boolean shouldSkipClass(Class<?> aClass) {
+      return aClass.getSimpleName().equalsIgnoreCase("ConsentGroup");
+    }
+  };
+
+  private final ExclusionStrategy consentGroupExclusionStrategy = new ExclusionStrategy() {
+    @Override
+    public boolean shouldSkipField(FieldAttributes fieldAttributes) {
+      final HashSet<String> exclusions = new HashSet<>(List.of(
+          "accessManagement",
```

Comment:
are data use fields an enum somewhere?

Reply:
Aha ... they are defined as static strings in `DatasetRegistrationSchemaV1Builder`, thank you! Will make that update.